### ISSUE TITLE
Undo/Redo does not respect QWERTZ keyboard layout

### ISSUE THREAD

**Issue by 5H4D0W-X** [Yellow-Dog-Man/Resonite-Issues#1387]
### Describe the bug?

When using a QWERTZ (German in my case) Keyboard, Pressing CTRL-Y triggers an undo and CTRL-Z triggers a redo, the opposite to the intended shortcuts.

### To Reproduce

I don’t have access to any QWERTY hardware keyboards, so I’m not sure if this works, but setting windows to a QWERTZ layout (so that pressing the hardware Y key prints a Z) might lead to the same problem (where pressing Z does an undo, even though it should do a redo since windows sees it as a Y key). If you have a QWERTY keyboard, please try this

### Expected behavior

CTRL-Z triggers the undo action, CRTL-Y triggers a redo

### Resonite Version Number

2024.2.12.1430

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Quest 2

### Log Files

N/A not an exception-based bug

### Additional Context

My Windows locale is set to German and Windows is configured to use a GER keyboard layout

### Reporters

ShadowX | @ shdw_x

**Comment by art0007i** [comment]
@shiftyscales  pretty sure this issue refers to dekstop mode and pressing ctrl + z on a physical QWERTZ keyboard, while the issue you mentioned refers to the VR keyboard that is used to type inside of text fields

**Comment by JackTheFoxOtter** [comment]
I'm... Legitimately surprised. I just figured this feature was broken, because pressing Ctrl+Z never did anything. Now I know better.

**Comment by Banane9** [comment]
So... I stumbled upon this issue and had to look into it. After some digging, I found that Resonite uses the `UnityEngine.InputSystem.Keyboard` to detect keypresses, which has behavior tailored to game control key mapping, as I found in a [Unity forum thread](https://forum.unity.com/threads/new-input-system-which-keyboard-layouts-are-supported.912074/).

TL;DR: The key names of the US keyboard layout are mapped to their position on the keyboard, which is then used to query that same position on the keyboard the user actually has. As Z and Y are swapped between QWERTZ and QWERTY, the displayed behavior is thus the expected behavior of this system. 
Additive text input is not affected by this, as it uses the `Keyboard.onTextInput` event, which directly provides the characters as printed on the keys.

They Keyboard class also provides the method `KeyControl FindKeyOnCurrentKeyboardLayout(string displayName)`, which 
provides the US layout key which is mapped to the key producing the given string on the user's actual keyboard (if it exists).
By querying this method for Z and Y, it should be easy to determine which US layout key needs to be queried, to get the appropriate pressed state.

Using this method would also be necessary for any other key bindings which are meant to be tied to the key as labeled, and not its position.
